tests/operators/test_zero_pad_op.py

```diff
@@ -12,34 +12,34 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-import numpy as np
 import pytest
 import torch
-
 from mrpro.operators import ZeroPadOp
+
 from tests import RandomGenerator
+from tests.helper import dotproduct_adjointness_test
 
 
 def test_zero_pad_op_content():
     """Test correct padding."""
```

Comment:
actually also tests cropping (i.e. negative padding). Suggestion for docstring: Test correct padding and cropping (i.e. negative padding size)